specs/description20230710caps.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Gauntlet Cap Updates for Ethereum, Optimism Aave v3
+author: Paul Lei, Watson Fu, Nick Del Zingaro, Walter Li, Jonathan Reem, Nick Cannon, Sarah Chen, Dana Tung, Aaruran Chandrasekhar
+discussions: [https://governance.aave.com/t/arfc-gauntlet-supply-cap-updates-for-ethereum-v3-optimism-v3-2023-07-05/13917]
+---
+
+## Summary
+
+Gauntlet recommends cap changes on Ethereum v3 and Optimism v3. For more details, see the full forum post [here]((https://governance.aave.com/t/arfc-gauntlet-supply-cap-updates-for-ethereum-v3-optimism-v3-2023-07-05/13917)).
```

Comment:
nit: link to forum post is broken in IPFS preview:

<img width="643" alt="Screenshot 2023-07-10 at 7 18 53 PM" src="https://github.com/GauntletNetworks/aave-proposals/assets/42387568/011fa5db-58d9-40b7-8ee8-db022d9d89f5">

Reply:
Updated: https://app.aave.com/governance/ipfs-preview/?ipfsHash=0x73fbf4d3b8dd87a2dd8d28ad0a6ca5b20a1767391950e07fe763d2d59d3af5ad